environment/src/test/java/com/sequenceiq/environment/environment/service/sdx/SdxUpgradeCcmPollerServiceTest.java

```diff
@@ -0,0 +1,39 @@
+package com.sequenceiq.environment.environment.service.sdx;
+
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import com.dyngr.core.AttemptResult;
+import com.dyngr.core.AttemptResults;
+import com.sequenceiq.environment.environment.poller.SdxPollerProvider;
+
+@ExtendWith(MockitoExtension.class)
+class SdxUpgradeCcmPollerServiceTest {
+
+    private static final String CRN = "crn";
+
+    private static final long ENV_ID = 123L;
+
+    @Mock
+    private SdxPollerProvider pollerProvider;
+
+    @InjectMocks
+    private SdxUpgradeCcmPollerService underTest;
+
+    @Test
+    void waitForUpgradeCcm() {
```

Comment:
Possibly add some further test cases for error scenarios.

Reply:
done